query-dsl/src/main/java/org/infinispan/query/dsl/impl/BaseQuery.java

```diff
@@ -54,6 +55,8 @@ protected BaseQuery(QueryFactory queryFactory, String queryString) {
       this.maxResults = -1;
    }
 
+   public abstract CloseableIterator<T> iterator();
```

Comment:
This override is not needed, the method comes from the interface already.

Reply:
yes. leftover